packages/members-api/lib/repositories/member.js

```diff
@@ -410,7 +423,8 @@ module.exports = class MemberRepository {
                     );
 
                     await this._StripeCustomerSubscription.upsert({
-                        status: updatedSubscription.status
+                        status: updatedSubscription.status,
+                        mrr: 0
```

Comment:
We can delete all of this at the end of the cycle FYI. When a member is deleted so are all of the related subscriptions/events/relations

Reply:
Is this already the case of will this change in 5.0?

Reply:
It's already the case, but let's leave it for cleanup IMO!